cves/CVE-2013-1665.yml

```diff
@@ -239,17 +278,26 @@ lessons:
     applies:
     note:
   native_wrappers:
-    applies:
-    note:
+    applies: true
+    note: |
+      The vulnerability involves developers trusting too much on the default security settings
+      of XML libraries and parsers. XML libraries and parsers support various
+      dangerous, unwidely-used features. Hence, developers should assume there are
+      always risks involved in using XML libraries and parsers and should practice
+      more input validations and sanitizations for instance.
   distrust_input:
     applies:
     note:
   security_by_obscurity:
     applies:
     note:
   serial_killer:
-    applies:
-    note:
+    applies: true
+    note: |
+      The vulnerability involves deserialization of XML, which could result in
+      potential confidentiality disclosure if it's not handled correctly. Also,
+      serialization is not encryption, so it's good to practice more defensive
```

Comment:
Great point about serialization/encryption

Reply:
Thank you!